modules/installing-rhv-configuring-network-infrastructure.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installing-rhv-configuring-network-infrastructure_{context}"]
+= Configuring the network infrastructure
+
+Before installing {product-title}, configure your network environment to meet the following requirements.
+
+When they boot, virtual machines must have IP addresses get the Ignition config files. Consider configuring DHCP to provide persistent IP addresses and hostnames to the cluster machines.
+// TBD - Day 0 versus day 2? Alternatives?
+
+.Firewall
+
+link:../../installing/install_config/configuring-firewall.html#configuring-firewall[Configure your firewall] so your cluster has access to required sites.
+
+.Network connectivity
+
+// TBD - What do we mean by "machine" here? Where do we configure this? Can this be done at this stage in the process?
+Configure your network to enable the following connections:
+
+* Grant every machine access to every other machine on ports `30000`-`32767`. This provides connectivity to {product-title} components.
+
+* Grant every machine access to reserved ports `10250`-`10259` and `9000`-`9999`.
+
+* Grant every machine access on ports `2379`-`2380`. This provides access to etcd, peers, and metrics on the control plane. Grant every machine access to the Kubernetes API on port `6443`.
+
+.Load balancers
+
+Configure one or two (preferred) layer-4 load balancers:
+
+* Provide load balancing for ports `6443` and `22623` on the control-plane and bootstrap machines (the bootstrap can be removed after control-plane initialization completes). Port `6443` provides access to the Kubernetes API server and must be reachable both internally and externally. Port `22623` must be accessible to nodes within the cluster.
```

Comment:
> Avoid using parentheses in running text because text in parentheses is decreased in importance.

In other UPI installations, you have to remove the bootstrap machine. Is that not true for RHV?